README.md

```diff
@@ -0,0 +1,50 @@
+# Alteration Effects module
+The Alteration Effects module contains a set of status effects (buffs and debuffs) that can alter a player's base state in some fashion.
+
+These are the `AlterationEffects` that current exist in the module:  
+- `HealthBoostAlterationEffect`- this increases the maximum health of the player by a specified amount 
```

Comment:
The "this" is redundant here (and in the entire list) IMO. "`HealthBoostAlterationEffect` - increases the maximum health of the player by a specified amount" sounds nicer 🙂